superset/connectors/sqla/models.py

```diff
@@ -1068,7 +1080,7 @@ def mutator(df: pd.DataFrame) -> None:
             """
 
             labels_expected = query_str_ext.labels_expected
-            if df is not None and not df.empty:
+            if df is not None:
```

Comment:
During testing I noticed that column names were not being renamed back to their original format if the dataframe was empty.

Reply:
Somewhat unrelated but I thought we banished `df` being `None`.